code/modules/projectiles/guns/smgs.dm

```diff
@@ -108,19 +108,26 @@
 		/obj/item/attachable/heavy_barrel,
 		/obj/item/attachable/scope/mini,
 		/obj/item/attachable/magnetic_harness,
+		/obj/item/attachable/gyro,
+		/obj/item/attachable/angledgrip,
 	)
 
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER|GUN_LOAD_INTO_CHAMBER
 	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_AUTOMATIC)
+	actions_types = list(/datum/action/item_action/aim_mode)
+	aim_fire_delay = 0.3 SECONDS //just dont
+	aim_speed_modifier = 2 //you wont be nearly as fast and mobile. kinda defeats the SMG main thing, unless you pack a gyro and then you loose on the Agrip
 	attachable_offset = list("muzzle_x" = 32, "muzzle_y" = 15,"rail_x" = 22, "rail_y" = 22, "under_x" = 17, "under_y" = 15, "stock_x" = 24, "stock_y" = 10)
 
 	accuracy_mult = 1.15
 	accuracy_mult_unwielded = 0.8
-	scatter = 0
-	fire_delay = 0.165 SECONDS
-	scatter_unwielded = 30
+	scatter = 5 //use the ext barrel for 2 hands a least buut i'm not your boss anyway
+	fire_delay = 0.132 SECONDS
+	scatter_unwielded = 25 //so you need the recoil comp and the gyro if shooting one handed. any other way you got scatter.
+	recoil_unwielded = 2 //you either get the recoil comp and gyro to one hand or you dont one hand  if you want to use suuupressor you got EVEN more damage fall of
 	aim_slowdown = 0.25
 	burst_amount = 0
+	damage_mult = 1.3 //should give that humpf it lacks. making the TTK better overall
```

Comment:
add a falloff multiplier so not every smg has doubled fallof as a result of this pr